### PR TITLE
Export spatialized data as CSV.

### PR DESCRIPTION
**Pull request by sylmak**

### CODE REVIEW

File: cloud_functions/climateiq_data_export_cf/main.py [hunk 1]
```diff
@@ -65,9 +65,13 @@ def export_model_predictions(cloud_event: http.CloudEvent) -> None:
         chunks_ref,
     )
 
-    # Set the error message to the Series' json string for testing
-    # purposes. In the actual implementation, the Series should be stored somewhere.
-    raise NotImplementedError(h3_predictions.to_json())
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
```
Comment (skeshive) [inline cloud_functions/climateiq_data_export_cf/main.py]: these predictions should be written to another bucket. 

so far we have:
climateiq-predictions --> stores ML raw predictions
climateiq-chunk-predictions --> stores ML raw predictions per chunk (and is read by this cloud function)

how about climateiq-spatialized-chunk-predictions for outputs?

File: cloud_functions/climateiq_data_export_cf/main.py [hunk 2]
```diff
@@ -65,9 +65,13 @@ def export_model_predictions(cloud_event: http.CloudEvent) -> None:
...
+    blob = bucket.blob(
+        f"{prediction_type}/{model_id}/{study_area_name}/{scenario_id}/{chunk_id}.csv"
+    )
+    with blob.open("w+") as fd:
+        h3_predictions.to_csv(fd)
```
Comment (skeshive) [inline cloud_functions/climateiq_data_export_cf/main.py]: was this schema confirmed with Jesus?

File: cloud_functions/climateiq_data_export_cf/main.py
Comment (skeshive) [inline cloud_functions/climateiq_data_export_cf/main.py:1]: also update comments for export_model_predictions()

File: cloud_functions/climateiq_data_export_cf/main.py [hunk 3]
```diff
@@ -65,9 +65,13 @@ def export_model_predictions(cloud_event: http.CloudEvent) -> None:
         chunks_ref,
     )
 
-    # Set the error message to the Series' json string for testing
-    # purposes. In the actual implementation, the Series should be stored somewhere.
-    raise NotImplementedError(h3_predictions.to_json())
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(
```
Comment (skeshive) [inline cloud_functions/climateiq_data_export_cf/main.py]: FYI we'll need to add the unique id from https://github.com/UrbanSystemsLab/climateiq-frontend/pull/22 here as well

File: cloud_functions/climateiq_data_export_cf/main.py (hunk 2 above)
Comment (sylmak) [reply cloud_functions/climateiq_data_export_cf/main.py]: Yes. The final schema (after merging) will be h3_index,scenario_0_id,scenario_1_id, etc. The schema for this intermediary step is just h3_index,prediction.

**Review by skeshive — APPROVED** [review]

**Pull request merged**